RdlMigration/RdlFileIO.cs

```diff
@@ -74,6 +74,18 @@ public bool IsFolder(string itemPath)
             }
         }
 
+        public bool IsFile(string itemPath)
```

Comment:
rename it to IsReport since that is what it checks

Reply:
done